modules/common/src/main/Form.scala

```diff
@@ -382,3 +382,12 @@ object Form:
       def unbind(key: String, value: Instant) =
         Map(key -> value.atZone(zone).toLocalDateTime.format(localFormatter))
     val mapping: Mapping[Instant] = of[Instant](format)
+
+  object SingleChange:
+    case class Change[Model, A](field: String, mapping: Mapping[A], update: A => Model => Model):
+      def form: PlayForm[A] = PlayForm(single(field -> mapping))
+
+    def changing[Model, FieldsType, A](field: FieldsType => (String, Mapping[A]))(
+        f: A => Model => Model
+    )(using fieldsType: FieldsType): Change[Model, A] =
+      Change(field(fieldsType)._1, field(fieldsType)._2, f)
```

Comment:
The whole partial update thing feels more convoluted and restrictive than it needs to be I think.

I've no concrete plan though, only thinking something using filled form etc would allow multiple changes at the same time, but no concrete plan.